orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -60,11 +61,34 @@ class CreateWebhookTask implements RetryableTask {
        response = webhookService.exchange(method, url, payload, customHeaders)
     } catch (HttpStatusCodeException e) {
       def statusCode = e.getStatusCode()
+
+      if (failFastStatusCodes != null) {
+        def isArray = [Collection, Object[]].any { it.isAssignableFrom(failFastStatusCodes.getClass()) }
```

Comment:
Any reason you didn't use `instanceOf`, as in `failFastStatusCodes instanceOf Collection`? That seems a little cleaner.

Reply:
There's a few cases where we're casing and whatnot ... alternative would be to `stage.mapTo()` a concrete type and just let jackson do the deserialization for you.

Reply:
Thanks! @ajordens pointed me in the direction for Jackson goodness, updating PR